chat-sdk/src/main/java/com/amazon/connect/chat/sdk/repository/ChatService.kt

```diff
@@ -590,12 +598,22 @@ class ChatServiceImpl @Inject constructor(
         return runCatching {
             val connectionDetails = connectionDetailsProvider.getConnectionDetails()
                 ?: throw Exception("No connection details available")
-            attachmentsManager.downloadAttachment(attachmentId, fileName, connectionDetails.connectionToken).getOrThrow()
+            attachmentsManager.downloadAttachment(connectionDetails.connectionToken, attachmentId, fileName).getOrThrow()
         }.onFailure { exception ->
             SDKLogger.logger.logError { "Failed to download attachment: ${exception.message}" }
         }
     }
 
+    override suspend fun getAttachmentDownloadUrl(attachmentId: String): Result<URL> {
+        return runCatching {
+            val connectionDetails = connectionDetailsProvider.getConnectionDetails()
+                ?: throw Exception("No connection details available")
+            attachmentsManager.getAttachmentDownloadUrl(attachmentId, connectionDetails.connectionToken).getOrThrow()
+        }.onFailure { exception ->
+            SDKLogger.logger.logError { "Failed to retrieve attachment download URL: ${exception.message}" }
```

Comment:
Does exception message include `attachmentId` ? if no, I think we should have it, for debugging purpose.

Reply:
It doesn't.  Good point - added IDs.